content/numerical/PolynomialPow.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "PolynomialBase.h"
+poly deriv(poly a) {
+	if (a.empty())
+		return {};
+	poly b(sz(a) - 1);
+	rep(i, 1, sz(a)) b[i - 1] = a[i] * num(i);
+	return b;
+}
+poly integr(poly a) {
+	if (a.empty()) return {0};
+	poly b(sz(a) + 1);
+	b[1] = num(1);
+	rep(i, 2, sz(b)) b[i] = b[mod%i]*Mod(-mod/i+mod);
```

Comment:
Doesn't work for complex, probably worth noting.

Reply:
Is it even worth noting stuff that doesn't work with complex numbers? Or is it reasonable to assume that every problem involving polynomials/series will be done on a finite field?

Reply:
Idk, the field of reals seems like it could be useful sometimes.